workspaces/default/themes/base/layouts/system/oauth2-redirect.html

```diff
@@ -0,0 +1,68 @@
+<!doctype html>
+<html lang="en-US">
+<title>Swagger UI: OAuth2 Redirect</title>
+<body onload="run()">
+</body>
+</html>
+<script>
+    'use strict';
+    function run () {
+        var oauth2 = window.opener.swaggerUIRedirectOauth2;
```

Comment:
Can we fix the spacing here to conform to our styling elsewhere? (2 space tabs)

Reply:
Preserving, as this will make keeping track of diffs from upstream easier.